wpimath/src/main/java/edu/wpi/first/math/controller/SimpleMotorFeedforward.java

```diff
@@ -150,7 +150,7 @@ public double calculate(double velocity, double acceleration) {
    *
    * @param velocity The velocity setpoint.
    * @return The computed feedforward.
-   * @deprecated Use the current/next velocity overload instead.
+   * @deprecated Use the unit-taking velocity overload instead.
```

Comment:
You should be able to link to the overload here

```suggestion
   * @deprecated Use {@link #calculate(Measure) the unit-taking velocity overload} instead
```